flexiblepricing/management/commands/configure_tiers.py

```diff
@@ -0,0 +1,283 @@
+        )
+        program_title = (
+            kwargs["program_name"] if "program_name" in kwargs else self.PROGRAM_TITLE
+        )
+        program_abbrev = (
+            kwargs["program_abbrev"]
+            if "program_abbrev" in kwargs
+            else self.PROGRAM_ABBREV
+        )
+
+        current_year = date.today().year
+        last_year = datetime(
+            current_year - 1, 1, 1, tzinfo=pytz.timezone(settings.TIME_ZONE)
+        )
+        discounts_and_tiers = [
+            {
+                "tier": {"threshold": 0},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier1-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 750,
+                },
+            },
+            {
+                "tier": {"threshold": 25000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier2-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 650,
+                },
+            },
+            {
+                "tier": {"threshold": 50000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier3-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 500,
+                },
+            },
+            {
+                "tier": {"threshold": 75000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier4-{current_year}",
+                    "discount_type": "percent-off",
+                    "amount": 0,
+                },
+            },
+        ]
+        content_type = ContentType.objects.filter(
+            app_label="courses", model="program"
+        ).first()
+
+        # Step zero: if the user supplied a defaults file, then parse that
+
+        if "tier_info" in kwargs and kwargs["tier_info"] is not None:
+            discounts_and_tiers = []
+            found_zero_tier = False
+
+            self.stdout.write("Reading tier info from file...")
+
+            with kwargs["tier_info"] as tierfile:
+                csvreader = csv.reader(tierfile)
+                for (idx, row) in enumerate(csvreader):
+                    self.stdout.write(
+                        f"Read tier: threshold {row[0]}, discount type {row[1]}, value {row[2]}"
+                    )
+                    discounts_and_tiers.append(
+                        {
+                            "tier": {"threshold": row[0]},
+                            "discount": {
+                                "discount_code": f"{program_abbrev}-fa-tier{idx+1}-{current_year}",
+                                "discount_type": row[1],
+                                "amount": row[2],
+                            },
+                        }
+                    )
+
+                    if int(row[0]) == 0:
+                        found_zero_tier = True
+
```

Comment:
With DictReader we could have this as:
```suggestion
                    self.stdout.write(
                        f"Read tier: threshold {row['threshold_amount']}, discount type {row['discount_type']}, value {row['discount_amount']}"
                    )
                    discounts_and_tiers.append(
                        {
                            "tier": {"threshold": row['threshold_amount']},
                            "discount": {
                                "discount_code": f"{program_abbrev}-fa-tier{idx+1}-{current_year}",
                                "discount_type": row['discount_type'],
                                "amount": row['discount_amount'],
                            },
                        }
                    )

                    if int(row['threshold_amount']) == 0:
                        found_zero_tier = True


```